batch_convert.qml

```diff
@@ -424,22 +428,57 @@ MuseScore {
     // other options
     property alias exportE: exportExcerpts.checked
     property alias travers: traverseSubdirs.checked
+    property alias diffEPath: differentExportPath.checked  // different export path
+    property alias ePath: targetFolderDialog.folder // export path
+    property alias iPath: sourceFolderDialog.folder // import path
     }
 
   FileDialog {
-    id: fileDialog
+    id: sourceFolderDialog
     title: traverseSubdirs.checked?
-      qsTr("Select Startfolder"):
-      qsTr("Select Folder")
+      qsTr("Select sources Startfolder"):
+      qsTr("Select sources Folder")
     selectFolder: true
+
     onAccepted: {
-      work(folder, traverseSubdirs.checked)
+      if(differentExportPath.checked){
```

Comment:
I dislike those superfluous braces and also the lack of whitespace (after `if`)

Reply:
About the coding style, I haven't run any sodeStyleChecker plugin because the indentation is quite messy in the whole plugin (sometimes it's 2 spaces, sometimes it's 4, and sometimes it's 3), and it would have marked the whole file as modified.
I will correct all these whitespace errors, but do you want me to clean the indentation too?

Reply:
No, leave the indent, except for that one you have already